### DISCUSSION TITLE
[mobile] Add possibility to force upload a image from mobile data despite only wifi is allowed

### DISCUSSION THREAD

**Discussion by Trekky12** [ente-io/ente#3294] [Enhancements]
It would be great to have the possibility to "force" upload a specific photo into a album while there is no wifi despite backup over mobile data is disabled.

**Comment by edrex** [comment]
Actually, I think the best use case for this would be to sync an album that you've created as a photo pool so that your receipients can see your shared images right away. This just bit me when i blasted out a pool link to a bunch of people at an event, and then of course I'll have to message them all again to check it when i get home and sync. I can avoid this in the future by waiting to share till i sync, but it's nice to have options.

I'd vote for a force sync item in the album context menu. But then that night require some job queue infra that night not exist yet 🤷

y'all are making a great product btw 🐦‍🔥
